db/src/db/api/modules/bitmap_translator.py

```
@@ -1,21 +1,31 @@
+LANG_TAGS = ["C", "Python", "Java", "Rust", "JavaScript", "Golang", "C++"]
+DIFF_TAGS = ["easy", "medium", "hard"]
+PROBLEM_TAGS = ["type #1", "type #2", "type #3", "type #4", "type #5"]
+
+TAGS = LANG_TAGS + DIFF_TAGS + PROBLEM_TAGS
+
+
 def translate_tags_to_bitmap(tags: list[str]) -> int:
     bitmap = 0
-
     for tag in tags:
-        if tag == "C":
-            bitmap += 1 << 0
-        elif tag == "python":
-            bitmap += 1 << 1
-
+        try:
+            index = TAGS.index(tag)
+            bitmap |= 1 << index
+        except ValueError:
+            continue
```

Should we silently continue after such an error? If this errors, it seems like an implementation problem that should be fixed before it causes (very hard to debug) errors beyond this function